routers/web/org/projects.go

```diff
@@ -600,7 +602,7 @@ func MoveIssues(ctx *context.Context) {
 		}
 	}
 
-	if err = project_model.MoveIssuesOnProjectColumn(ctx, column, sortedIssueIDs); err != nil {
```

Comment:
It seems that `project_model.MoveIssuesOnProjectColumn` isn't removed.

Reply:
f1b942e